web2/htdocs/shield.css

```diff
@@ -865,7 +865,7 @@ form .ctl label {
   line-height: 20px;
   display: block;
 
-  width: 40%;
+  width: 50%;
```

Comment:
How well does this play with the error reporting on form validation?

Reply:
Seems to work okay, errors still appear as they used to